mkt/search/forms.py

```diff
@@ -71,7 +72,10 @@
     ('featured-game-adventure', 'featured-game-adventure'),
     ('featured-game-puzzle', 'featured-game-puzzle'),
     ('featured-game-strategy', 'featured-game-strategy'),
+    ('featured-website', 'featured-website'),
 ]
+REG_TAG = lambda r: ('featured-website-%s' % r, 'featured-website-%s' % r)
```

Comment:
Seemed like it'd be more readable this way if it had to be multiple lines, but maybe not. I've been writing a lot of JS lately.